src/GenSymIO_broken.chpl

```diff
@@ -0,0 +1,642 @@
+module GenSymIO {
```

Comment:
@reuster986 should this file be deleted?

Reply:
Yes, will delete.